railties/test/generators/app_generator_test.rb

```diff
@@ -914,8 +877,9 @@ def test_css_option_with_sass_uses_cssbundling_gem_when_using_node
 
   def test_css_option_with_cssbundling_gem
     run_generator_and_bundler [destination_root, "--css=postcss"]
+
+    assert_includes @rails_commands, "css:install:postcss", "`css:install:postcss` expected to be called, but wasn't."
```

Comment:
Same comment here regarding test coverage in the `cssbundling-rails` gem and changing `run_generator_and_bundler` to `run_generator`.